schemas/main.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
This file may be better expressed as a simple `curl` request. I elected to write it in Go so that it would be easier to alter/expand upon, but perhaps that's premature optimization.